core/domain/email_manager.py

```diff
@@ -1159,6 +1207,113 @@ def send_mail_to_notify_users_to_review(user_id, category):
+                'There were no suggestions to recommend to the reviewer with '
+                'user id: %s.' % reviewer_id)
+            continue
+        elif not reviewer_emails[index]:
+            log_new_error(
+                'There was no email for the given reviewer id: %s.' % (
+                    reviewer_id))
+            continue
+        else:
+            list_of_suggestions_to_notify_reviewer_strings = []
+            for reviewer_suggestion_email_info in (
+                    reviewers_suggestion_email_infos[index]):
+                # Get the language of the suggestion.
+                language = utils.get_supported_audio_language_description(
+                    reviewer_suggestion_email_info.language_code)
+                # Calculate how long the suggestion has been waiting for review.
+                suggestion_review_wait_time = (
+                    datetime.datetime.utcnow() - (
+                        reviewer_suggestion_email_info.submission_datetime))
+                # Get a string composed of the largest time unit that has a
+                # value, followed by that time unit. For example, if the
+                # suggestion had been waiting for review for 5 days and 2 hours,
+                # '5 days' would be returned. This is more user friendly since a
+                # high level of precision is not needed.
+                human_readable_review_wait_time = (
+                    utils.create_string_from_largest_unit_in_timedelta(
+                        suggestion_review_wait_time))
+                values_to_populate_suggestion_template_dict = {
+                    'language': language,
+                    'review_wait_time': human_readable_review_wait_time,
+                    'suggestion_content': (
+                        reviewer_suggestion_email_info.suggestion_content)
+                }
+                get_values_to_populate_suggestion_template = (
+                    NOTIFY_CONTRIBUTOR_DASHBOARD_REVIEWERS_EMAIL_INFO[
+                        'suggestion_template_values_getter_functions'][
+                            reviewer_suggestion_email_info.suggestion_type])
+                suggestion_template = (
+                    NOTIFY_CONTRIBUTOR_DASHBOARD_REVIEWERS_EMAIL_INFO[
+                        'suggestion_template'][
+                            reviewer_suggestion_email_info.suggestion_type])
+                list_of_suggestions_to_notify_reviewer_strings.append(
+                    suggestion_template % (
+                        get_values_to_populate_suggestion_template(
+                            values_to_populate_suggestion_template_dict)))
+
+            email_body = email_body_template % (
+                reviewer_usernames[index], feconf.OPPIA_SITE_URL,
+                feconf.CONTRIBUTOR_DASHBOARD_URL, ''.join(
+                    list_of_suggestions_to_notify_reviewer_strings),
+                EMAIL_FOOTER.value)
+
+            _send_email(
```

Comment:
Should we run this in translation? (Also, it's in for loop so that would be helpful.)

Reply:
Ah, I see. Thanks for the explanation!